src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -43,11 +43,13 @@
   {# A tiny helper pixel to detect if we've scrolled #}
   <div id="pst-scroll-pixel-helper"></div>
 
+  {%- if html_back_to_top_button %}
```

Comment:
```suggestion
  {%- if theme_back_to_top_button %}
```
For me locally the button was *never* shown unless I make this change.

Reply:
And I was banging my head against the wall not understanding why.... I'm very bad with typos.
thanks !